### PR TITLE
li/hw1

### PR DESCRIPTION
**Pull request by lilyreber**

### CODE REVIEW

**Comment by yana-tanch** [comment]
1) train_correct, val_correct = 0, 0
   train_total, val_total = 0, 0 
накапливаются на протяжении всех эпох, а не обнуляются каждую эпоху -> неправильное вычисление accuracy

2) Двойной вызов
plot.update({
        "loss": train_loss,
        "val_loss": val_loss
    })

    plot.update({
        'loss': train_loss,
        'val_loss': val_loss,
        'accuracy': train_acc,
        'val_accuracy': val_acc
    })

3) Модель загружается на CPU, а данные могут быть на GPU
model.load_state_dict(state_dict, strict=True)  

4) На тестовых данных не хватает вычисления и вывода точности

**Comment by lilyreber** [comment]
Все комментарии учла

**Review by yana-tanch — APPROVED** [review]
